tasks/server.js

```diff
@@ -0,0 +1,16 @@
+const gulp = require('gulp');
+const server = require("browser-sync").create();
+
+gulp.task("server", function () {
```

Comment:
there's some quote inconsistency in this file, please use single quotes for this string and the one on line 2.